src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx

```diff
@@ -59,16 +59,13 @@ export interface FlyoutProps {
   done: () => void;
   newIndexPatternUrl: string;
   indexPatterns: IndexPatternsContract;
-  overlays: OverlayStart;
   http: HttpStart;
   basePath: IBasePath;
   search: DataPublicPluginStart['search'];
 }
 
 export interface FlyoutState {
-  conflictedIndexPatterns?: any[];
-  conflictedSavedObjectsLinkedToSavedSearches?: any[];
-  conflictedSearchDocs?: any[];
+  conflictedSearchDocs?: any[]; // potentially unused
```

Comment:
I think it is, I don't see any usages outside of `resolve_saved_objects`. Any issue when you tried to remove it?